src/schedule/ll_schedule.c

```diff
@@ -684,6 +684,11 @@ static int schedule_ll_task_cancel(void *data, struct task *task)
 	return 0;
 }
 
+#if CONFIG_SCHEDULE_LL_NO_RESCHEDULE_TASK
+/* As a null function pointer */
+#define reschedule_ll_task ((void*)0)
```

Comment:
why not just `NULL`?

Reply:
Both ways are probably fine. The reason I chose `((void*)0)` is because `NULL` in c is also a macro, and I'm not sure if there is any risk when a macro represents as another macro.

Reply:
I'd be very surprised if this didn't work. We certainly have macros using macros like across a dozen or more layers. And this is in a C file, so it won't be included in other files to potentially be used in further macro definitions.